lib/rubocop/cli.rb

```diff
@@ -52,7 +52,7 @@ def run(args = ARGV)
       STATUS_ERROR
     rescue StandardError, SyntaxError, LoadError => e
       warn e.message
-      warn e.backtrace
+      warn 'For usage information, use --help'
```

Comment:
This would remove the stack trace for a host of other issues though, in addition to invalid command line parameters?

Reply:
Good catch. Will update.